app/forms/gpo_verify_form.rb

```diff
@@ -37,7 +37,7 @@ def submit
         enqueued_at: gpo_confirmation_code&.code_sent_at,
         pii_like_keypaths: [[:errors, :otp], [:error_details, :otp]],
         pending_in_person_enrollment: pending_in_person_enrollment?,
-        threatmetrix_check_failed: fraud_check_failed?,
+        threatmetrix_check_failed: fraud_review_check.fraud_check_failed?,
```

Comment:
Also tangential to this PR but closer in scope - I wonder if PersonalKeyController can do the fraud_review_check in a before action to decide whether to show the "You're verified!" msg, and take all the fraud-related code out of GPOVerifyForm/Controller entirely.

Reply:
Also, PersonalKeyController has `blocked_by_device_profiling?` which is reaching directly into the profile to check fraud review status. Can that go through FraudReviewConcern as well?